src/nodes/function_expression.js

```diff
@@ -34,6 +34,10 @@ export function format(node, context, recur, {noFunctionKeyword} = {}) {
         context.write('function');
     }
 
+    if (node.generator) {
+        context.write('* ');
```

Comment:
To match this one https://github.com/dmitriiabramov/esfmt/commit/f362b8546aa879a8b27d40774d07a8520746905e#diff-926a5400e756f7ac901a5cc4b77695f0R14